backend/scripts/lint.sh

```
@@ -1,6 +1,12 @@
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
```

Shellcheck (0.9.0-1)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)